Btw, @drodriguez, any idea why [11 C++ Interoperability tests fail on the Android CI](https://ci-external.swift.org/job/oss-swift-RA-linux-ubuntu-16.04-android-arm64/789/console)? Seems like it's looking in the wrong place, ie the host linux headers, wonder if you've looked into it:
```
/usr/include/stdlib.h:25:10: error: 'bits/libc-header-start.h' file not found
```
I have fixes ready for two other tests that were failing recently on the Android CI that I will submit, but I see 18 more tests that are failing today.

Also, could you get my pull #59001 in? I asked Karoy, but it looks like he's off github this week.

--- a/test/CMakeLists.txt
+++ b/test/CMakeLists.txt
@@ -219,6 +219,7 @@ set(TEST_MODES
     optimize_none_with_implicit_dynamic
     optimize_with_implicit_dynamic
     only_executable only_non_executable
+    with_cxx_interop
 )
 set(TEST_SUBSETS
     primary

--- a/test/lit.cfg
+++ b/test/lit.cfg
@@ -708,6 +708,10 @@ elif swift_test_mode == 'only_executable':
     config.limit_to_features.add("executable_test")
 elif swift_test_mode == 'only_non_executable':
     config.available_features.add("nonexecutable_test")
+elif swift_test_mode == 'with_cxx_interop':
+    config.available_features.add("with_cxx_interop")
+    config.swift_frontend_test_options += ' -enable-experimental-cxx-interop'
+    config.swift_driver_test_options += ' -Xfrontend -enable-experimental-cxx-interop'
 else:
     lit_config.fatal("Unknown test mode %r" % swift_test_mode)
 

